hello-world-mtls/src/exec-workflow.ts

```diff
@@ -0,0 +1,48 @@
+import fs from 'fs';
```

Comment:
I'd call this 'starter.ts' to be consistent with other examples and other docs.

Reply:
I think `starter.ts` might be ambiguous to a newbie—just looking at the files, maybe it's starting a worker or activity:

![image](https://user-images.githubusercontent.com/251288/132373795-f51d5c0e-7451-461d-a823-482631d4717b.png)

What do you think about:

```
  "scripts": {
    ...
    "start-workflow": "ts-node src/start-workflow.ts"
  }
```

or 

```
  "scripts": {
    ...
    "execute-workflow": "ts-node src/execute-workflow.ts"
  }
```

These make it clear what's getting started/executed. The latter matches the function that's being called (`stub.execute`, not `stub.start`, at least in this example), and I think feels more in line with the fact that the script runs and completes, vs the `"start"` worker script that stays running until killed. Maybe we could always do the latter, or always use the verb that matches the stub function.

Reply:
I don't know if we need a `start-workflow` script.
I kind of like calling the script `workflow` so it can be called with `npm run workflow`.

Reply:
I'm fine with that.

```
  "scripts": {
    ...
    "workflow": "ts-node src/execute-workflow.ts"
  }
```